cmake/deps/libxrpl.cmake

```diff
@@ -1 +1 @@
-find_package(xrpl REQUIRED CONFIG)
+find_package(xrpl-mpt REQUIRED CONFIG)
```

Comment:
I think this should be reverted before merging

Reply:
Yes these will be changed when it's ready to merge. We'd need wait for the rippled one to merge first